ccp/src/prover/tests.rs

```diff
@@ -97,17 +98,34 @@ fn load_state(state_dir: &Path) -> Option<CCPState> {
     serde_json::from_slice(&state_data).unwrap()
 }
 
+async fn wait_for_proofs(
+    prover: &CCProver,
+    length: usize,
+    retries: usize,
+    retry_delay: Duration,
+) -> eyre::Result<Vec<CCProof>> {
+    for _retry in 0..retries {
+        tokio::time::sleep(retry_delay).await;
+        let proofs = prover.get_proofs_after("0".parse().unwrap()).await?;
+        if proofs.len() > length {
```

Comment:
Do you have a case in mind where `proofs.len() > length`?

Reply:
I don't understand what do you mean.